python/ground/common/model/core/edge.py

```diff
@@ -0,0 +1,44 @@
+from ..version.item import Item
+
+
+class Edge(Item):
```

Comment:
There should also be a constructor that takes in the actual fields. The client methods should be able to take in these objects as arguments to the `create` methods. e.g., `create_edge(edge)`.

Reply:
Line 4 isn't defining a constructor, it's defining the `Edge` class as extending the `Item` class. On line 13 there's a classmethod `from_edge` which creates a new edge from a previous edge object (usage would be `new_edge = Edge.from_edge(id, old_edge)`. Should that be changed to use `old_edge`'s id (so usage could be `Edge.from_edge(old_edge)`, and they would both share the same id)?